src/test/java/org/broadinstitute/consent/http/service/InstitutionServiceTest.java

```diff
@@ -191,6 +194,93 @@ void testFindAllInstitutions() {
     assertTrue(service.findAllInstitutions().isEmpty());
   }
 
+  @Test
+  void testCreateInstitutionDomainUniquenessAllUnique() {
+    Institution mockInstitution = initMockModel();
+    mockInstitution.setDomains(List.of("broadinstitute.org", "broad.mit.edu"));
```

Comment:
Add testing for IDNs

```suggestion
    mockInstitution.setDomains(List.of("broadinstitute.org", "broad.mit.edu", "café.com"));
```

Reply:
Good call!